moonbeam-types-bundle/README.md

```diff
@@ -3,20 +3,74 @@
 Exports npm package `moonbeam-types-bundle`, formated as per polkadot-js specification to use
 with the app or the api.
 
-## Exports
+## ⚠️Warning: Types deprecation⚠️
 
+Following runtime upgrade 900 (include substrate v0.9.11), types are now retrieved from the node, in
+a **camelCase** format.
```

Comment:
... and there is no need to inject types in the api from that version on.

Reply:
Well you still need to inject them (but PolkadotJS is doing it) if you query blocks before runtime 900.